modules/meteroid/Cargo.toml

```diff
@@ -79,17 +79,18 @@ jsonwebtoken.workspace = true
 
 once_cell.workspace = true
 tower-layer.workspace = true
+fastrand = { workspace = true }
 
 [dev-dependencies]
 rstest = { workspace = true }
+svix.workspace = true
```

Comment:
for now we use it only in tests to verify the generated webhook secret, we will use svix for signing requests so it will be moved into non-dev deps in next PR.